hs_explore/migrations/0002_auto_20200625_1910.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-06-25 19:10
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def fill_csdms_names(apps, schema_editor):
+    """ Store CSDMS names into LDAWord model
+    """
+    LDAWord = apps.get_model('hs_explore', 'LDAWord')
+    CSDMSName = apps.get_model('hs_csdms', 'CSDMSName')
+    csdms_names = set()
+    splitted_names = set()
+    for csdms_record in CSDMSName.objects.filter(part='name'):
```

Comment:
Same thoughts for this, there should be an LDAWord for every CSDMSName, yet we've got custom code in a migration to do the sync once.